testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -2297,10 +2326,13 @@ QUnit.module('keyboard navigation', {
 
         prevented = 0;
         this.instance.option('opened', true);
+        assert.strictEqual(this.instance.option('opened'), true);
```

Comment:
Why did u add this asserts? It seems to be excess here

Reply:
i was thinking it'd be nice to check if popup is opened before closing it, but yeah, don't think we should check if 'opened' option works correctly, sure it is already tested

Reply:
I added separate test for testing esc button -  'escape key should close popup'